pkg/polaris/creater.go

```diff
@@ -196,6 +199,22 @@ func GetPolarisBaseSecrets(polaris Polaris) (map[string]runtime.Object, error) {
 	return mapOfUniqueIDToBaseRuntimeObject, nil
 }
 
+func GetVersions(baseURL string) ([]string, error) {
+	serverURL, err := url.Parse(baseURL)
+	if err != nil {
+		return nil, err
+	}
+
+	serverURL.Path = path.Join(serverURL.Path, "polaris", "VERSIONS")
+
+	content, err := util.HTTPGet(serverURL.String())
+	if err != nil {
+		return nil, err
```

Comment:
again, please increase log verbosity

Reply:
The error returned by HTTPGet is already verbose enough. What more information do you want?

Reply:
I guess verbosity is the wrong word, maybe the right word is context.  This is just gonna say 404 not found, or can't talk to the server.  We should make it clear to the user that this is in regards to the context of not being able to fetch the yaml chart :)

Reply:
The error message has been updated.